.github/workflows/ci.yml

```diff
@@ -4,12 +4,12 @@ on: [push, pull_request]
 
 jobs:
   build-macos:
-    runs-on: macos-latest
+    runs-on: macOS-11
```

Comment:
`macOS-10.15` is no longer supported

Reply:
It seems like `macos-latest` was already macOS 11, and soon/now will be macOS 12? https://github.blog/changelog/2022-10-03-github-actions-jobs-running-on-macos-latest-are-now-running-on-macos-12/

Curious why you think it's best to switch to an explicit version.

Reply:
Maybe it has to do with Xcode?

Reply:
When we originally wrote this `ci.yml` file, `macos-latest` was equivalent to `macOS-10.15`.

As the macOS runner changes, so does the Xcodes available in CI. Running CI on `macos-latest` requires CI to be independent of the Xcode version. Since that's not the case here (we care about what Xcode version we're running CI on), I chose a specific macOS runner.